docs/docs/contributing.md

```diff
@@ -16,37 +19,47 @@ channel in Slack, hang out in there and see if there are any questions you
 can help with.
 
 You might also want to keep an eye on the
-[OPA Discussion Board](https://github.com/orgs/open-policy-agent/discussions).
+[OPA Discussion Board](https://github.com/orgs/open-policy-agent/discussions) or
+on the [`#rego`](https://stackoverflow.com/questions/tagged/rego) tag on Stack
+Overflow.
 
 ## I'd like to contribute code
 
-If you have found a bug and would like to work on a fix, we still encourage you
-file a [GitHub Issue](https://github.com/open-policy-agent/opa/issues) first
-to talk about the problem and the solution you have in mind.
+If you have found a bug and would like to work on a fix, **we always encourage you
+file a [GitHub Issue](https://github.com/open-policy-agent/opa/issues)** to talk
+about the problem and the solution you have in mind. This allows you to get
+feedback from maintainers before committing time to something that might already
+have a solution or might not be the right approach.
 
-Similarly, if you have an idea for a new feature, we encourage you to file an
-issue to discuss the feature before you start working on it too.
+If you have an idea for a new feature, we also **request that you file an
+issue** to discuss it first. This again allows you to get feedback from
+the maintainer team and the community before you start working on it.
```

Comment:
Should we also add a paragraph about first creating an issue in `.github/PULL_REQUEST_TEMPLATE.md`?
It already has a link to here, so maybe that's enough ..